en/db-models.md

```diff
@@ -474,7 +474,35 @@ Query the first record that matches the specified conditions. It will return a r
 
 use MyApp\Models\Invoices;
 
-$invoice  = Invoices::findFirst();
+$invoice = Invoices::findFirst();
+```
+
+> **NOTE**: If primary key of table is not numeric, use condition. See examples below.
+ {: .alert .alert-warning }
+
+```php
+<?php
+
+
+use MyApp\Models\Invoices;
+
+$invoice = Invoices::findFirst('uuid = "5741bfd7-6870-40b7-adf6-cbacb515b9a9"');
```

Comment:
Please add note here:
> If you use constants instead of bound parameters in your PHQL it creates **a new planning** consuming **more memory**.

Reply:
`If you do not use bound parameters in your conditions, PHQL will create a new plan internally, therefore consuming **more memory**. Using bound parameters is **highly recommended**!